apps/discordbot/bot.py

```diff
@@ -37,16 +42,33 @@ async def on_ready(self):
             self.test_channel = get(self.test_guild.channels, id=TEST_CHANNEL_ID)
             self.hoser_role = get(self.csua_guild.roles, id=HOSER_ROLE_ID)
 
+    async def verify_member_email(self, user): 
+        channel = user.dm_channel
+        def check_msg(msg, user):
+            return msg.channel == channel
+        got_email = False
+        while not got_email:
+            msg = await self.wait_for('message', check=check_msg)
+            try:
+                validate_email(msg)
+                got_email= True
+                await channel.send(f"Sending a an email to verify {user.name} to {msg}")
+                send_verify_mail(msg, user.name)
+            except ValidationError as e:
+                await channel.send(f"{msg} is not a valid email. Please try again. Details: ", e)
+    
     async def on_message(self, message):
         if message.author == self.user:
             return
+
+        #Reading rules and verification
+
+
         msg = message.content.lower()
-        if msg == "piss":
-            await message.channel.send("shid")
-        elif msg == "poo" or msg == "poop":
-            await message.channel.send("funny poopies")
-        elif msg == "uh oh":
-            await message.channel.send("stinky")
+        if "hkn" in msg.lower() and "ieee" in msg.lower():
```

Comment:
Did you mean `if "hkn" in msg or "ieee" in msg:`?

Reply:
Also, nice.

Reply:
I don't want it to trigger it accidentally, so only if both clubs are mentioned will it trigger